client/src/utils/api-calls/auth.js

```diff
@@ -3,7 +3,6 @@ import axios from "axios";
 export const loginUser = async (username, password, dispatch) => {
 
     try {
-    
         const response = await axios.post(
           `https://apirestream.sohamsshah.repl.co/auth/login`,
```

Comment:
You can create a constant for API URL and then use it here, so that in case you want to update API URL it becomes easy to change only at one place.